### ISSUE TITLE
Add error-specific download handling

### ISSUE THREAD

**Issue by RumovZ** [bohning/usdb_syncer#40]
If a resource is age or geo restricted, the appropriate action should be taken.

**Comment by bohning** [comment]
Good test case for age restriction (both YT and Vimeo): http://usdb.animux.de/index.php?&link=detail&id=16397

**Comment by randompersona1** [comment]
I feel this might need to be part of a ui addition/redesign. The log is too full to show error messages like this that require manual intervention by the user. Perhaps a completely new section underneath the log intended for more user-friendly instructions on what to do?

**Comment by RumovZ** [comment]
I agree that the log is too stuffed to be a reliable way to communicate with the user, at least when you download multiple songs at once. But I don't think yet another wall of text is the solution. We should put more thought into which log level is supposed to serve which purpose.
